.travis.yml

```diff
@@ -1,25 +1,47 @@
 language: php
 
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
   - 7.0
   - hhvm
- 
+
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.5.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2"
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=1.3.*
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=2.0.*@dev
+  allow_failures:
+    - php: hhvm
+    - env: SYMFONY_VERSION=2.8.*@dev
+    - env: SYMFONY_VERSION="3.0.x-dev as 2"
```

Comment:
See failing tests that __can't__ resolve such version (ps. my comment should be few lines above were you declare this env).